catalog/airbyte/x-definitions/app-airbyte.cue

```diff
@@ -0,0 +1,112 @@
+"airbyte": {
+	annotations: {}
+	labels: {}
+	attributes: {
+		apiResource: {
+			definition: {
+				apiVersion: "bdc.kdp.io/v1alpha1"
+				kind:       "Application"
+				type:       "airbyte"
+			}
+		}
+	}
+	description: "airbyte"
+	type:        "xdefinition"
+}
+
+template: {
+	output: {
+		apiVersion: "core.oam.dev/v1beta1"
+		kind:       "Application"
+		metadata: {
+			name:      context["name"]
+			namespace: context["namespace"]
+		}
+		spec: {
+			components: [
+				{
+					name: context["name"]
+					properties: {
+						chart:           "airbyte"
+						releaseName:     context["name"]
+						repoType:        "oci"
+						targetNamespace: context["namespace"]
+						url:             context["helm_repo_url"]
+						values: {
+							webapp: {
+								replicaCount: 1
+							}
+							server: {
+								replicaCount: 1
+							}
+							worker: {
```

Comment:
Shall we at least expose the replica setting of worker, when a user needs higher TPS?

Reply:
more configuration exposed  for airbyte components